README.md

```diff
@@ -44,8 +44,70 @@ The keypairs are RSA with 2048 bit private keys.
       signed with the "tpm" key. A UKI signed with the "tpm" key permit access to
       the TPM password for administration.
 
+# Entities / Ownership
+**Michael** (think microsoft) owns the firmware.
+  * owns keys: uefi-pk, uefi-kek
+  * public keys for uefi-pk and uefi-kek are the UEFI Platform key and UEFI Key Exchange Key respectively.
+  * In secure boot mode, the firmware measures into PCR-7 the signer of an EFI application.
+
+**Mikey** owns a key that is in the system's UEFI DB.
+  * owns keys: uefi-db
+  * uefi-db here is a single key that is present in the uefi-db.
+  * EFI applications signed by uefi-db will be executed by the UEFI in SecureBoot mode.
+  * Mikey signs Steven's shim with Mikey's uefi-db key.
+
+**Steven** owns the shim.efi application:
+  * owns keys: uki-limited, uki-production, uki-tpm
+  * it is assumed that Steven has sole access to those 3 keys.
+  * Steven's 3 public keys are present in the shim's built-in DB.
+  * The shim's primary purpose is to validate the kernel that it will execute, ensuring that it is signed by one of the keys in it's DB.
+  * The shim measures the signer of the efi application (kernel) into PCR-7
+
+**Arthur** is the Manifest Certificate Authority.
+ * owns keys: manifest-ca
+ * As the Manifest Certificate Authority (manifest-ca), Arthur signs Oliver's Certificate Signing Request (manifest).
+
+**Katherine** owns the kernel.
+  * owns keys: uki-limited, uki-production, uki-tpm.
```

Comment:
Per your comments, both Katherine and Steven "own" the uki-limited keys.

So I think you just want to say, above, that Steven owns the shim, and put's the certificates for Katherine's keys in the shim.

Reply:
+1